tests/Backend/CommonPart1/ExportParamsTest.php

```diff
@@ -204,14 +206,16 @@ public function testTableExportAsyncPermissions()
         $this->assertFalse($results['cacheHit']);
         $this->waitForFile($fileId);
 
-        $newTokenId = $this->_client->createToken(array(
-            $this->getTestBucketId() => 'read',
-        ));
+        $tokenOptions = (new TokenCreateOptions())
+            ->addBucketPermission($this->getTestBucketId(), TokenUpdateOptions::BUCKET_PERMISSION_READ)
+        ;
+
+        $newTokenId = $this->_client->createToken($tokenOptions);
         $newToken = $this->_client->getToken($newTokenId);
-        $client = new \Keboola\StorageApi\Client(array(
+        $client = new \Keboola\StorageApi\Client([
```

Comment:
ten namespace je zbytečnej, už to je v `use`

Reply:
fixed